pyproject.toml

```diff
@@ -17,14 +17,15 @@ dependencies = [
     "seaborn>=0.13.2",
     "plotly>=5.23.0",
     "trimesh>=4.1.0",
+    "vtk==9.3.0",
```

Comment:
@mfs4rd is `vtk` used? I don't see it in any of the imports.

Reply:
`vtk` is a dependency within `pyvista`, this is pinning a version that is compatible with the code in `inside-outside_classification.py`